tutorials/hsdk-ngc-container/README.md

```diff
@@ -0,0 +1,99 @@
+# Holoscan SDK VS Code DevContainer Template
```

Comment:
Would you please add a `metadata.json` for this HoloHub project? You can copy from the template at https://github.com/nvidia-holoscan/holohub/blob/main/tutorials/template/metadata.json.template

Reply:
Done